docs/Contribute-to-AlmaLinux-Build-System.md

```diff
@@ -0,0 +1,187 @@
+* Add description of config to `roles/dev_deploy/defaults/main/configs.yml`.
+
+#### A new directory
+* Add description to `roles/dev_deploy/defaults/main/common.yml`.
+
+#### A new docker container
+* Add a new description of a container's docker image to `roles/dev_deploy/defaults/main/docker_images.yml`.
+* Add a new description of a docker container to `roles/dev_deploy/defaults/main/docker_containers.yml`.
+
+## Commit Guidelines
+
+When you are ready to create a pull request with your suggested changes, we'd like to provide some commit guidelines in order to make our commit messages consistent across all repositories around the AlmaLinux Build System. Every contributor could use this document to make commits that:
+* Make sense (grammatically).
+* Are meaningful/clear enough -> describe the change(s) made to the code.
+* Include references to Github issue (whenever possible).
+* Include references to fixed (or resolved) issue(s).
+* This guide also aims to cover not only atomic commits made by contributors but also how merge commits must be performed.
+
+### Commit messages
+For this first iteration, we aim to focus on these main two goals:
+* Commit messages must be precise about the changes made so other developers understand what happened in that commit.
+* A person can identify the ticket in Github that motivated such change. If possible link a GitHub issue that the commit(s) fixed or resolved. 
+
+**Remember**, these goals apply to both regular and merge commits.
+
+### Structure
+Depending on the context, a commit message might have specific needs in different situations, i.e.:
+
+* There is only a single commit that fixes a single issue.
+* When there is a list of meaningul commits that fixes one or several issues at a time.
+* When there's no issue involved.
+
+Even if we don't want to be very pedantic about how commits look like (unless we decide to strengthen the policy for other reasons), we'd like to be more or less consistent with the goals described above.
+
+#### A very detailed commit message
+
+The commit message should meet the requirements:
+* Should contain short (72 chars or less) summary.
+* Should contain more detailed explanatory text. Wrap it to 72 characters. The blank line separating the summary from the body is critical (unless you omit the body entirely).
+* Further paragraphs come after blank lines.
+* Bullet points are okay, too.
+    * Typically a hyphen or asterisk is used for the bullet, followed by a single space. Use a hanging indent.
+* If want to add references to fixed or resolved issue(s), do it at the end as in the exmaple: 
+  ```
+  Fixes: https://github.com/AlmaLinux/build-system/issues/80.
+  ```
+  or 
+  ```
+  Resolves: https://github.com/AlmaLinux/build-system/issues/80.
+  ```
+
+Ideally, every commit message should look like this, but we try to be flexible enough.
```

Comment:
I'd love to see us link to any commit message that incorporates all of these guidelines (especially if the example shows a list being used in addition to linking to an existing issue), so folks can see a real-world implementation of this guide.

Reply:
Here's one https://github.com/AlmaLinux/albs-web-server/commit/db062d4e69fe7bc93c59ce5c4c977e3fc3667419